OrbitLinuxTracing/TestPuppetLibrary.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <cmath>
+
+// extern "C" to have unmangled name in the symbol table
+extern "C" double function_that_that_works_for_considerable_amount_of_time();
```

Comment:
Can't you just add `extern "C"` to the function definition instead of having a separate declaration?

Reply:
done.